javascript/javascript2/week1/homework/hyf-bay/main.js

```diff
@@ -0,0 +1,26 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+function renderProducts(products) {
+    // your code here
+    const productList = document.querySelector("ul");
```

Comment:
Consider moving this const outside the function. Each call to `querySelector` is slow so if you save the const outside the function you can reuse it. That way, `querySelector` is only called once.

Reply:
Thanks for the comment. I'll do that.